### PR TITLE
charts,salt: Bump NGINX Ingress chart to v4.8.4

### PR DESCRIPTION
**Pull request by ezekiel-alexrod**
Bump NGINX Ingress chart to v4.8.4 and also bump the Ingress Controller image to v1.9.4

Upgrade NGINX Ingress chart using:
```
rm -rf charts/ingress-nginx
helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
helm repo update
helm fetch -d charts --untar ingress-nginx/ingress-nginx
```

Re-render NGINX Ingress salt states using:
```
./doit.sh codegen:chart_ingress-nginx
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -77,6 +77,12 @@
   and `metalk8s-keepalived` image to `alpine:3.18.5`
   (PR[#4208](https://github.com/scality/metalk8s/pull/4208))
 
+- Bump ingress-nginx chart version to
+  [4.8.4](https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-4.8.4)
+  The controller image has been bumped accordingly to
+  [v1.9.4](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.9.4)
+  (PR[#4211](https://github.com/scality/metalk8s/pull/4211))
```
Comment (TeddyAndrieux) [inline CHANGELOG.md]: It was already bumped in this release so just update the previous changelog entry (just 2 above this one) to match the new version and PR

Comment (ezekiel-alexrod) [reply CHANGELOG.md]: Whoopsie, you're right

File: CHANGELOG.md [hunk 2]
```diff
@@ -77,6 +77,7 @@
   and `metalk8s-keepalived` image to `alpine:3.18.5`
   (PR[#4208](https://github.com/scality/metalk8s/pull/4208))
 
+
```
Comment (TeddyAndrieux) [inline CHANGELOG.md]: :thinking:

**Review by TeddyAndrieux — APPROVED** [review]

**Pull request merged**